.github/workflows/ci.yml

```diff
@@ -258,6 +258,8 @@ jobs:
 
       - name: Setup Boost
         run: ci\github\install.bat
+        env:
+          B2_BOOTSTRAP_TOOLSET: ${{ matrix.bootstrap-toolset }}
```

Comment:
Maybe add the same to the Posix part (at https://github.com/boostorg/boost-ci/pull/137/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR201)?

Reply:
Okay now I see what you meant. :)